pkg/agent/openflow/fields.go

```diff
@@ -102,20 +129,25 @@ var (
 	// reg4[0..18]: Field to store the union value of Endpoint port and Endpoint status. It is used as a single match
 	// when needed.
 	EpUnionField = binding.NewRegField(4, 0, 18, "EndpointUnion")
-	// reg4[19]: Field to mark that whether Service type is NodePort.
-	NodePortAddressField = binding.NewRegField(4, 19, 19, "NodePortAddress")
-	// ToNodePortAddressRegMark marks that the Service type as NodePort.
-	ToNodePortAddressRegMark = binding.NewRegMark(NodePortAddressField, 0b1)
-	// reg4[20]: Field to mark that whether the packet of Service NodePort/LoadBalancer from gateway requires SNAT.
-	ServiceSNATField = binding.NewRegField(4, 20, 20, "ServiceSNAT")
-	// ServiceNeedSNATRegMark marks that the packet of Service NodePort/LoadBalancer requires SNAT.
-	ServiceNeedSNATRegMark = binding.NewRegMark(ServiceSNATField, 0b1)
+	// reg4[19]: Mark to indicate the Service type is NodePort.
+	ToNodePortAddressRegMark = binding.NewOneBitRegMark(4, 19, "NodePortAddress")
 	// reg4[16..19]: Field to store the union value of Endpoint state and the mark of whether Service type is NodePort.
 	NodePortUnionField = binding.NewRegField(4, 16, 19, "NodePortUnion")
-	// reg4[21]: Mark to indicate the packet is from local AntreaFlexibleIPAM Pod.
-	// NotAntreaFlexibleIPAMRegMark will be used with RewriteMACRegMark, thus the reg id must not be same due to the limitation of ofnet library.
-	AntreaFlexibleIPAMRegMark    = binding.NewOneBitRegMark(4, 21, "AntreaFlexibleIPAM")
-	NotAntreaFlexibleIPAMRegMark = binding.NewOneBitZeroRegMark(4, 21, "AntreaFlexibleIPAM")
+	// reg4[20]: Field to indicate whether the packet is from local Antrea IPAM Pod. NotAntreaFlexibleIPAMRegMark will
+	// be used with RewriteMACRegMark, thus the reg id must not be same due to the limitation of ofnet library.
+	AntreaFlexibleIPAMRegMark    = binding.NewOneBitRegMark(4, 20, "AntreaFlexibleIPAM")
+	NotAntreaFlexibleIPAMRegMark = binding.NewOneBitZeroRegMark(4, 20, "AntreaFlexibleIPAM")
```

Comment:
the name `AntreaFlexibleIPAM` should be `NotAntreaFlexibleIPAM`?

Reply:
Yes, thanks.